### PR TITLE
Add Brands-related action + reducer

### PR DESCRIPTION
**Pull request by joshiefishbein**

### CODE REVIEW

File: src/reducers/data/brands.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { FETCH_BRANDS } from '../../actions/data/brands';
+
+export const initialState = {};
+
+export default (state = initialState, action) => {
+  const { payload } = action;
+
+  switch (action.type) {
+    case `${FETCH_BRANDS}_FULFILLED`:
```
Comment (sepowitz) [inline src/reducers/data/brands.js]: We need to track this action in ```error``` and ```status``` reducers

File: src/actions/data/brands.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import fireAction from '../../utils/fireAction';
+import handleErrors from '../../utils/handleErrors';
+
```
Comment (sepowitz) [inline src/actions/data/brands.js]: let's add some simple test coverage this action

File: src/reducers/data/brands.js (hunk 1 above)
Comment (sepowitz) [reply src/reducers/data/brands.js]: let's add some simple test coverage for this reducer

File: src/actions/data/brands.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export const FETCH_BRANDS = 'FETCH_BRANDS';
```
Comment (sepowitz) [inline src/actions/data/brands.js]: This is a ```show``` so we should rename this to ```FETCH_BRAND```

File: package.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
   "author": "Sanctuary Computer <hello@sanctuary.computer> (http://sanctuary.computer/)",
   "license": "MIT",
   "dependencies": {
-    "brandibble": "git+https://git@github.com/sanctuarycomputer/brandibble.js#bc91c55cc062ead325ee3571c06a4875990d7ab3",
+    "brandibble": "git+https://git@github.com/sanctuarycomputer/brandibble.js#a11aadb6b75b76174fabb391c88baa7ab3bf52b7",
```
Comment (hhff) [inline package.json]: Now that master is merged on brandibble.js can we update to point at the latest SHA?

**Pull request merged**